example/src/components/ChatboxTest.tsx

```diff
@@ -102,78 +113,95 @@ const ChatboxTest: FC = () => {
+            label="Chat Id"
+            labelPlacement="top"
           />
-          <FormControlLabel
-            control={
-              <Slider
-                value={testWidth}
-                min={GRAASP_PANEL_WIDTH}
-                step={20}
-                max={800}
-                color="secondary"
-                onChange={onChangePanelWidth}
+          <FormControl>
+            <FormLabel component="legend">Language</FormLabel>
+            <RadioGroup
+              aria-label="language"
+              value={lang}
+              onChange={({ target }): void => setLang(target.value)}
+            >
+              <FormControlLabel value="fr" control={<Radio />} label="French" />
+              <FormControlLabel
+                value="en"
+                control={<Radio />}
+                label="English"
               />
-            }
-            labelPlacement="top"
-            label="Panel Width"
+            </RadioGroup>
+          </FormControl>
+          <FormControl>
+            <FormLabel component="legend">Chatbox params</FormLabel>
+            <FormControlLabel
+              control={
+                <Checkbox
+                  value={showTools}
+                  onChange={(): void => setShowTools(!showTools)}
+                />
+              }
+              label="Show Admin tools"
+            />
+            <FormControlLabel
+              control={
+                <Slider
+                  value={testWidth}
+                  min={GRAASP_PANEL_WIDTH}
+                  step={20}
+                  max={800}
+                  color="secondary"
+                  onChange={onChangePanelWidth}
+                />
+              }
+              labelPlacement="top"
+              label="Panel Width"
+            />
+          </FormControl>
+          <MentionButton
+            color="primary"
+            useMentions={hooks.useMentions}
+            useMembers={hooks.useMembers}
+            patchMentionFunction={patchMentionFunction}
+            deleteMentionFunction={deleteMentionFunction}
+            clearAllMentionsFunction={clearAllMentionsFunction}
           />
-        </FormControl>
-        <MentionButton
-          color="primary"
-          useMentions={hooks.useMentions}
-          useMembers={hooks.useMembers}
-          patchMentionFunction={patchMentionFunction}
-          deleteMentionFunction={deleteMentionFunction}
-          clearAllMentionsFunction={clearAllMentionsFunction}
-        />
-      </div>
-      <div className={classes.chatboxContainer}>
-        <ChatboxWrapper
-          chatId={chatId}
-          lang={lang}
-          showAdminTools={showTools}
-        />
+        </div>
+        <div className={classes.chatboxContainer}>
+          <ChatboxWrapper
+            chatId={chatId}
+            lang={lang}
+            showAdminTools={showTools}
+          />
+        </div>
       </div>
-    </div>
+    </I18nextProvider>
   );
 };
```

Comment:
I look at this file and wonder if it would match well with storybook? 🤔

Reply:
@pyphilia indeed you are right, it would match well with storybook to test the different props and parameters and then be able to interact with it. I will open an issue to tackle that.